src/AWSAuth.jl

```diff
@@ -1,9 +1,9 @@
 module AWSAuth
 
-export AWS4AuthRequest, AWS4AuthLayer
+export SignatureV4, AWS4AuthLayer
 
-include("AWS4AuthRequest.jl")
+include("signaturev4.jl")
```

Comment:
"signature_v4.jl"?

Reply:
Sure, I just figured it made sense for the file name to match the name of the module it contains. (But lowercase, because uppercase file names that aren't the main module definition file are a pet peeve of mine.)